Button/Button.module.less

```diff
@@ -4,6 +4,16 @@
 @import "../styles/variables.less";
 @import "../styles/skin.less";
 
+@keyframes scale {
```

Comment:
Where is this used? I'm missing the reference

Reply:
It isn't used. Was considering using in other skins or in an animation bank. It doesn't technically need to be in this PR.